src/librbd/image/CreateRequest.cc

```diff
@@ -71,7 +71,7 @@ int validate_striping(CephContext *cct, uint8_t order, uint64_t stripe_unit,
     lderr(cct) << "must specify both (or neither) of stripe-unit and "
                << "stripe-count" << dendl;
     return -EINVAL;
-  } else if (stripe_unit || stripe_count) {
+  } else if ( stripe_unit ) {
```

Comment:
(1) don't insert unnecessary spaces and (2) if you are going to make this trivial change, why not then collapse the nested if statement at the same time?